x-pack/plugins/triggers_actions_ui/public/types.ts

```diff
@@ -135,15 +136,10 @@ export const OPTIONAL_ACTION_VARIABLES = ['context'] as const;
 export type ActionVariables = AsActionVariables<typeof REQUIRED_ACTION_VARIABLES[number]> &
   Partial<AsActionVariables<typeof OPTIONAL_ACTION_VARIABLES[number]>>;
 
-export interface AlertType {
-  id: string;
-  name: string;
-  actionGroups: ActionGroup[];
+export interface AlertType extends Omit<AlertTypeApi, 'actionVariables'> {
```

Comment:
**question:** since this `AlertType` doesn't related directly with the common alert type, should we pick or omit attributes? Just thinking in the scenario a new property is added to the common alert type.

Reply:
Yeah, I think Pick will be better here